dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsViewModel.ts

```diff
@@ -59,12 +64,15 @@ export class EuTaxonomyForNonFinancialsViewModel implements FrameworkViewModel {
         nfrdMandatory: apiModel.general?.nfrdMandatory,
       },
     };
-    this.assurance = {
-      assurance: {
-        levelOfAssurance: apiModel.general?.assurance?.assurance ?? AssuranceDataAssuranceEnum.None,
-        assuranceProvider: apiModel.general?.assurance?.provider,
-      },
-    };
+    this.assurance =
+      apiModel.general?.assurance?.assurance == undefined
+        ? undefined
```

Comment:
Can someone explain this change to me?

Reply:
It seems to me that this code change avoids the subcategory "Assurance" from being displayed in the frontend, if the field "apiModel.general?.assurance?.assurance" is undefined in the backend data.
That way it probably avoids "Subcategory -> Subcategory" to be shown in the frontend, without any field underneath it.

Reply:
Thank you. A bit of a weird work around, but ok for now.